assets/js/components/user-input/UserInputPreview.js

```diff
@@ -0,0 +1,128 @@
+/**
+ * User Input Preview.
+ *
+ * Site Kit by Google, Copyright 2020 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/**
+ * External dependencies
+ */
+import PropTypes from 'prop-types';
+
+/**
+ * WordPress dependencies
+ */
+import { __ } from '@wordpress/i18n';
+import { useCallback, Fragment } from '@wordpress/element';
+
+/**
+ * Internal dependencies
+ */
+import Data from 'googlesitekit-data';
+import { STORE_NAME as CORE_USER } from '../../googlesitekit/datastore/user/constants';
+import { Cell, Row } from '../../material-components';
+import Button from '../button';
+import ProgressBar from '../ProgressBar';
+import UserInputPreviewGroup from './UserInputPreviewGroup';
+import UserInputQuestionNotice from './UserInputQuestionNotice';
+import {
+	USER_INPUT_ANSWERS_GOALS,
+	USER_INPUT_ANSWERS_HELP_NEEDED,
+	USER_INPUT_ANSWERS_POST_FREQUENCY,
+	USER_INPUT_ANSWERS_ROLE,
+} from './util/constants';
+const { useSelect, useDispatch } = Data;
+
+export default function UserInputPreview( { goTo } ) {
+	const settings = useSelect( ( select ) => select( CORE_USER ).getUserInputSettings() );
+	const {
+		hasStartedSavingInputSettings,
+		hasFinishedSavingInputSettings,
+	} = useSelect( ( select ) => ( {
+		hasStartedSavingInputSettings: select( CORE_USER ).hasStartedSavingInputSettings(),
+		hasFinishedSavingInputSettings: select( CORE_USER ).hasFinishedSavingInputSettings(),
+	} ) );
+
+	const { saveUserInputSettings } = useDispatch( CORE_USER );
+	const submitChanges = useCallback( () => {
+		saveUserInputSettings( settings );
+	}, [ settings ] );
+
+	return (
+		<div className="googlesitekit-user-input__preview">
+			<Row>
+				<Cell>
+					{ hasStartedSavingInputSettings && ! hasFinishedSavingInputSettings && (
```

Comment:
See above, this should be `isFetchingSaveUserInputSettings || isNavigating`.

Reply:
Decided to stick with `hasStartedSavingInputSettings` and `hasFinishedSavingInputSettings` because it better helps to identify the status of the process.